jaxrs/WFLY-15695-resteasy-6.1-upgrade.adoc

```diff
@@ -0,0 +1,131 @@
+= Upgrade RESTEasy from 4.x to 6.1
+:author:            James R. Perkins
+:email:             jperkins@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+RESTEasy 6.1 is Jakarta REST 3.1 (Jakarta EE 10) compliant. The MicroProfile REST Client, MicroProfile REST Config
+Sources and Spring were moved out of the project and into new projects in RESTEasy 5.x. This was due to the project
+dependencies not being Jakarta EE 10 compliant.
+
+One new module will be introduced for the `org.jboss.resteasy.microprofile:microprofile-config`.
+
+Two new projects:
+
+* https://github.com/resteasy/resteasy-microprofile
+* https://github.com/resteasy/resteasy-spring
+
+|===
+|Old GA |New GA
+
+|`org.jboss.resteasy:resteasy-client-microprofile`
+|`org.jboss.resteasy.microprofile:microprofile-rest-client`
+
+|`org.jboss.resteasy:resteasy-client-microprofile-base`
+|`org.jboss.resteasy.microprofile:microprofile-rest-client-bas`
```

Comment:
Could it be missing the last 'e'? `so org.jboss.resteasy.microprofile:microprofile-rest-client-base`

Reply:
Nice catch @yersan. I'll fix it now, thank you!